integration/setup.go

```diff
@@ -220,3 +220,17 @@ func startup(t *testing.T) {
 
 	go debug.RunHandler(ctx, "127.0.0.1:0", zap.L().Named("debug"))
 }
+
+// dbByPort returns database name by port.
+func dbByPort() string {
```

Comment:
I see ports hardcoded in `cmd/envtool/main.go`, so I guess it's ok that we hard-code them here as well...
I believe, one day we might need to move them to consts/vars, but for now it seems to be ok.